t/15-assets.t

```diff
@@ -120,6 +121,9 @@ is($job->{id}, $jobA->id, 'jobA grabbed');
 is(scalar @assets, 1,         'job still has only one asset assigned after grabbing');
 is($assets[0],     $theasset, 'the assigned asset is the same');
 
+# assume worker picked up the job
```

Comment:
I tend to favor `note` here to have kind of "executable comments" which can also help to read the log output.